start-testnet-client.sh

```diff
@@ -21,9 +21,9 @@ curl -O https://enclave-distribution.test.mobilecoin.com/${SIGSTRUCT_URI}
 
 TARGETDIR=./target/release
 
```

Comment:
The file name no longer matches what it does since there’s no test net client

Reply:
It's still technically a testnet client - just using mobilecoind-json as the client, with the connection to testnet hardcoded.